.travis.yml

```diff
@@ -57,7 +57,13 @@ matrix:
     - env: TARGET=armv7-apple-ios NORUN=1
 script:
   - rustup target add $TARGET || true
-  - cargo build --target $TARGET
+  - cargo build --target $TARGET --verbose 2>&1 | tee build_std.txt
+  - cargo build --no-default-features $TARGET --verbose 2>&1 | tee build_no_std.txt
+
+  # Check that the no-std builds do not include std by accident:
+  - cat build_std.txt | grep -q "default" && cat build_std.txt | grep -q "use_std"
+  - ! cat build_no_std.txt | grep -q "default" && ! cat build_no_std.txt | grep -q "use_std"
```

Comment:
Might be easier / more water-tight to use something like

```
$ nm path/to/rlib | grep '_Z3std'
```

(I *think* `nm` works on rlibs, but it might complain about the extra metadata inside them)

Reply:
Yeah, that's what I had in mind. I'll test this locally and will do that in a different PR (will fill an issue about this to not forget it).